### ISSUE TITLE
About NoisyNN pretrained weights

### ISSUE THREAD

**Issue by pavaris-pm** [kytimmylai/NoisyNN-PyTorch#2]
Since this is an unofficial implementation of NoisyNN, which means that we do not have a pretrained model weight. All we have is an estimated architecture. If we need to achieve the nearest result according to the paper. We need to pretrained it with the full pretrained dataset right?

Or you already made some experiment from partial samples of it? Is it work ?

**Comment by kytimmylai** [comment]
Hi,

The pretrain weight is usually inherited from the previous work. We typically don't retrain it.

I care about the 95% accuracy on Imagenet w/ ViT-B, so other experiments may not be considered.
